scene/gui/patch_9_frame.cpp

```diff
@@ -95,6 +95,20 @@ void Patch9Frame::set_patch_margin(Margin p_margin,int p_size) {
 	margin[p_margin]=p_size;
 	update();
 	minimum_size_changed();
+    switch (p_margin) {
```

Comment:
Looks like you used spaces for indentation instead of tabs, could you have a look at fixing that?